src/KafkaInstanceStatusPopover/KafkaInstanceStatusPopover.stories.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { KafkaInstanceStatusPopover } from "./KafkaInstanceStatusPopover";
+import { InstanceStatus } from "./KafkaInstanceStatusProgressStepper";
+
+export default {
+  title: "Components/KafkaStatusPopover",
+  component: KafkaInstanceStatusPopover,
+  args: {
+    isOpen: true,
+    isVertical: true,
+  },
+  followsDesignSystem: true,
+  parameters: {
+    docs: {
+      description: {
+        component: `
+  Introduction/Background
+
+  This is a popover that can be reached from the Kafka Instance table while the status of the instance in question is “Creating”.
+  A Kafka instance usually takes a few minutes to create. The purpose of this popover is to give the user some insights into
+  the creation process of their instance while they wait for it to be available to use.
+
+  -The text at the top explains that the instance will be ready shortly & suggests that the user creates a service account while they wait.
+   (Provides a link to the “Connections tab" where this is done).
+
+  -The steps shown in the progress stepper are statuses that show in the api. They have been grouped together under the one overall
+  “Creating status” in the table.There is some descriptive text accompanying each step too. When these steps are complete the instance will be ready for use
+
+  -The popover is only available while instance creation is in progress.
+
+  -After creation is complete the table displays the ready status and the popover is no longer available.
+
+  Pending Status
```

Comment:
@riccardo-forina - do you know if we can use md to format the contents in this file, so that the dashes "-" display as bullets and the "Pending Status" can display as a heading?

Reply:
I think you can, yes!